src/test/mockResultTableStore.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Any reason for organizing/locating this file as `src/test/mockResultTableStore.ts`?